frontend/src/lib-customizations/espoo/citizen/assets/i18n/en.tsx

```diff
@@ -1558,6 +1560,73 @@ const en: Translations = {
       deleteConfirm: 'Haluatko poistaa tuloselvityksen?',
       deleteDescription:
         'Haluatko varmasti poistaa toimittamasi tuloselvityksen? Kaikki poistettavan lomakkeen tiedot menetetään.'
+    },
+    view: {
+      title: 'Tuloselvityslomake',
```

Comment:
I'm probably just unaware of our current practices but why are these full Finnish translations in the English file and not either totally empty (typings?) or some obvious placeholders?

Reply:
Translations are pending (external translator).

Reply:
Yeah but what I'm after is that there's an inherent risk in using real translations as placeholders: can't probably count on one hand the times I've forgotten about updating some translations because I didn't realize they were placeholders after-the-fact.

But if this is our current practice, ignore this nit. Just something to keep in mind.

Reply:
This https://github.com/espoon-voltti/evaka/pull/1332/commits/9a9de2e3d9d7509f96216f8d7165c0aee8432ad8 should work for now?